.bazelrc.common

```diff
@@ -0,0 +1,26 @@
+# Bazel settings that apply to this repository.
```

Comment:
may as well include the other standards like strict action env

Reply:
yeah; we should align all of our rulesets and create tooling to keep things like this consistent across repos